spring-cloud-kubernetes-integration-tests/spring-cloud-kubernetes-configuration-watcher-it/src/test/resources/kafka/kafka-deployment.yaml

```diff
@@ -54,4 +51,4 @@ spec:
             # that is set automatically. Since we don't have that many, producers will fail.
             # This setting ensures that there is just one replication
             - name: KAFKA_OFFSETS_TOPIC_REPLICATION_FACTOR
-              value: 1
```

Comment:
also this must be a string. I only noticed since I was reusing some things from here at my workplaces (which recently is eating all of my time)